ui/src/common/PlayButton.js

```diff
@@ -3,28 +3,50 @@ import PropTypes from 'prop-types'
 import PlayArrowIcon from '@material-ui/icons/PlayArrow'
 import { IconButton } from '@material-ui/core'
 import { useDispatch } from 'react-redux'
+import { useDataProvider } from 'react-admin'
+import { playTracks } from '../audioplayer'
 
-const defaultIcon = <PlayArrowIcon fontSize="small" />
-
-const PlayButton = ({ icon = defaultIcon, action, ...rest }) => {
+const PlayButton = ({ record, size = 'small', ...rest }) => {
```

Comment:
Color should be forced to `white`, to make it always visible over an album cover. Or make it a parameter (as it is in the `ContextMenu`), so it can be used in other places?
<img width="258" alt="Screen Shot 2020-10-23 at 9 12 49 AM" src="https://user-images.githubusercontent.com/331353/97009182-c9d8fb80-1511-11eb-8e57-c733af793378.png">